### DISCUSSION TITLE
github.dev is broken on my iPad

### DISCUSSION THREAD

**Discussion by robertandrews** [community/community#67161] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When accessing github.dev directly or by pressing ‘.’ in a repo, loading stops at “Setting up your web editor”. The [same problem affects vscode.dev](https://github.com/microsoft/vscode/issues/190930).

This is impacting my iPad Pro 12.9 on iPadOS 15.6.1, both Safari and Chrome. It used to work on the same setup, but now now. No known changes have been made since the last time it worked.

The editor loads okay on iPhone and Mac at the same location, on the same network.

Things tried:

* Private mode
* Disabling Content Blockers/Extensions
* Clearing Safari browser data for github.dev and vscode.dev

**Comment by P-mistry** [comment]
Try to Chatgpt past your code so chahgpt find a problems or bug in your code

**Reply by robertandrews** [reply]
The problem has nothing to do with my code.

**Comment by luqmanlani** [comment]
are the bugs fixed now? i'm having the same problem with the same device.

**Reply by mezotv** [reply]
Not fixed yet, no.

**Comment by mhanuel26** [comment]
@robertandrews, @luqmanlani 

I can confirm this was also the case on my iPad 12.9 Pro with iOS 16 but once I update to iOS 17 the issue is gone. 

Cheers,
Manuel

**Comment by gabfr** [comment]
whenever i try to use two tabs with github.dev of different projects/repo for instance, and try to copy paste files between each other, the first file works, then the copy works but the paste, doesnt, it pastes a weird formatted version of the file (like url encoded)…

and when I use paste to preserve the format it doesnt works, simply does *nothing*

**Comment by xrootme** [comment]
Im same ho to fix it
